doc/manuals/admin/logs.md

```diff
@@ -224,6 +224,7 @@ Alarm conditions:
 | 3          | CRITICAL   | The following ERROR text appears in the 'msg' field: "Raising alarm DatabaseError: `<detail>`"                    | The following ERROR text appears in the 'msg' field: "Releasing alarm DatabaseError". Orion prints this trace when it detects that DB is ok again."                                                                                       | Database Error. The `<detail>` text contains the detailed information.                                        | Orion is unable to access MongoDB database and/or MongoDB database is not working properly. Check database connection and database status. Once the database is repaired and/or its connection to Orion, the problem should disappear (Orion service restart is not needed). No specific action has to be performed at Orion Context Broker service.
```

Comment:
An entry about the changes included in this PR, citing issue #1794, should be included in CHANGES_NEXT_RELEASE file.

Reply:
Added entry about the changes in CHANGES_NEXT_RELEASE file.